src/briefcase/platforms/linux/system.py

```diff
@@ -994,6 +1017,104 @@ def _package_rpm(self, app: AppConfig, **kwargs):
+        if not changelog_source.is_file():
+            raise BriefcaseCommandError(
+                """\
+Your project does not contain a CHANGELOG file.
+Create a file named `CHANGELOG` in the same directory as your `pyproject.toml`
+with details about the release.
+"""
+            )
+
+        # Generate the pkgbuild layout
+        pkgbuild_path = self.bundle_path(app) / "pkgbuild"
+        with self.input.wait_bar("Generating pkgbuild layout..."):
+            if pkgbuild_path.exists():
+                self.tools.shutil.rmtree(pkgbuild_path)
+            pkgbuild_path.mkdir(parents=True)
+
+            # Copy the CHANGELOG file to the pkgbuild_path so that it is visible to PKGBUILD
+            self.tools.shutil.copy(changelog_source, pkgbuild_path / "CHANGELOG")
+
+        # Build the source archive
+        with self.input.wait_bar("Building source archive..."):
+            self.tools.shutil.make_archive(
+                pkgbuild_path / f"{app.app_name}-{app.version}",
+                format="gztar",
+                root_dir=self.bundle_path(app),
+                base_dir=f"{app.app_name}-{app.version}",
+            )
+
+        # Write the arch PKGBUILD file.
+        with self.input.wait_bar("Write PKGBUILD file..."):
+            # Add runtime package dependencies. App config has been finalized,
+            # so this will be the target-specific definition, if one exists.
+            system_runtime_requires_list = [
+                f"glibc>={app.glibc_version}",
+                "python3",
+            ] + getattr(app, "system_runtime_requires", [])
+
+            system_runtime_requires = " ".join(
+                f"'{pkg}'" for pkg in system_runtime_requires_list
+            )
+
+            with (pkgbuild_path / "PKGBUILD").open("w", encoding="utf-8") as f:
+                f.write(
+                    "\n".join(
+                        [
+                            f"# Maintainer: {app.author} <{app.author_email}>",
+                            f'export PACKAGER="{app.author} <{app.author_email}>"',
+                            f"pkgname={app.app_name}",
+                            f"pkgver={app.version}",
+                            f"pkgrel={getattr(app, 'revision', 1)}",
+                            f'pkgdesc="{app.description}"',
+                            f"arch=('{self.tools.host_arch}')",
+                            f'url="{app.url}"',
+                            f"license=('{app.license}')",
+                            f"depends=({system_runtime_requires})",
+                            "changelog=CHANGELOG",
+                            'source=("$pkgname-$pkgver.tar.gz")',
+                            "md5sums=('SKIP')",
+                            "package() {",
+                            '    cp -r "$srcdir/$pkgname-$pkgver/usr/" "$pkgdir"/usr/',
+                            "}",
+                        ]
+                    )
+                )
```

Comment:
(creating discrete comment from review)

`makepkg` is eager to strip the binaries in the package which is likely to cause problems for libraries from PyPI wheels; this was also a problem for RPM packaging. Adding `options=('!strip')` to `PKGBUILD` will disable all stripping.
```suggestion
            with (pkgbuild_path / "PKGBUILD").open("w", encoding="utf-8") as f:
                f.write(
                    "\n".join(
                        [
                            f"# Maintainer: {app.author} <{app.author_email}>",
                            f'export PACKAGER="{app.author} <{app.author_email}>"',
                            f"pkgname={app.app_name}",
                            f"pkgver={app.version}",
                            f"pkgrel={getattr(app, 'revision', 1)}",
                            f'pkgdesc="{app.description}"',
                            f"arch=('{self.tools.host_arch}')",
                            f'url="{app.url}"',
                            f"license=('{app.license}')",
                            f"depends=({system_runtime_requires})",
                            "changelog=CHANGELOG",
                            'source=("$pkgname-$pkgver.tar.gz")',
                            "md5sums=('SKIP')",
                            "options=('!strip')",
                            "package() {",
                            '    cp -r "$srcdir/$pkgname-$pkgver/usr/" "$pkgdir"/usr/',
                            "}",
                        ]
                    )
                )
```